README.md

```diff
@@ -56,6 +58,15 @@ conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults \
     cudf=0.6 python=3.6 cudatoolkit=10.0
```

Comment:
I think you mean `cudf=0.7` here?

Reply:
Corrected this with commit : https://github.com/rapidsai/cudf/pull/1657/commits/c44bc9adf3e537e1af512b68cad726d1826b733f

Reply:
I prefer explicit values in-line here.. less chance to copy/paste incorrectly

Reply:
Added explicit values for 0.6 and 0.7.